### DISCUSSION TITLE
Display a single number out side of a data table

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2143] [Help] [answered]
I would like to display the total number of rows that have been submitted, I have created its own view that has just one row and one column that calculates the number from the SQL table. I would like to show this number in line with a headline however I can not get the handlebars call to work. the syntax I have tried for this is {{table_name.Column_name}}.

The best I have been able to do is display this value in a single row single-column table is there a better way to do this?
![image](https://user-images.githubusercontent.com/80058238/127224132-a25c6d5c-666e-4c0a-a5cf-bee00312bb93.png)

**Comment by shogunpurple** [comment, marked as answer]
Hi @AndrewJRamay - you should be able to use a repeater to do this instead of the table.

- Create a repeater component and bind it to your data provider
- Put a container or headline inside it
- Bind the value of the headline to the `{{table_name.Column_name}}.`

Let me know if this helps!

**Reply by KernOil** [reply]
@shogunpurple that was the step I was missing! they all need to be subsets of each other 
1 data provider 
2 repeater 
3 headline 
I did not have the headline under the repeater. makes sense now you say it. Thank you very much
